schema/object_test.go

```diff
@@ -552,7 +552,7 @@ func TestObjectSchema_ValidateCompatibility(t *testing.T) {
 	assert.Error(t, testStructSchema.ValidateCompatibility(testOptionalFieldSchema)) // Not the same
 	// Schema validation with ref
 	objectTestRef := schema.NewRefSchema("testStruct", nil)
-	objectTestRef.ApplyScope(testStructScope, schema.DEFAULT_NAMESPACE)
+	objectTestRef.ApplyNamespace(testStructScope.Objects(), schema.SelfNamespace)
```

Comment:
Should this be using `ApplySelf()`?  (Otherwise, should the first argument be `nil`?)

Reply:
The left type is a Ref, not a scope. So no; it's correct.